Toolset/palettes/inspector/editors/com.livecode.pi.customprops.behavior.livecodescript

```diff
@@ -113,8 +113,12 @@ on editorResize
    send "groupResize kLabelSize, kControlGap" to group "Set buttons" of me
    
    # Use all space not taken by other elements for main array display
-   local tWidgetHeight
-   put the height of me - (4 * kControlGap + kLabelFieldHeight + kKeyFieldHeight + kValueFieldHeight + kSetButtonsHeight) into tWidgetHeight
+      local tWidgetHeight, tValueHeight
+   put 150 into tWidgetHeight -- or other value
```

Comment:
Could you use a named constant for this `150` so that they are all defined in the same place? `kWidgetHeight` would be fine :-)

Reply:
done